src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/RazorSemanticTokensInfoServiceTest.cs

```diff
@@ -60,7 +63,7 @@ public async Task GetSemanticTokens_CSharpBlock_HTML()
         Assert.NotEmpty(csharpTokens.Tokens);
     }
 
-    [Fact]
+    [Fact(Skip = "https://github.com/dotnet/razor/issues/9312")]
     public async Task GetSemanticTokens_CSharp_Nested_HTML()
```

Comment:
I thought we could enable this without the endpoint being available? The bump here I thought would only be for our testing, as the version that ships with VS would be P3

Reply:
the test is skipped because the behavior for the skipped tests is different when they use the original endpoint, they provide better classification in these cases causing a baseline file mismatch.